<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Nehas-MBP.attlocal.net" name="bioinformatics_practice.rosalind.WrightFisherModelTest" tests="7" failures="1" timestamp="15 Aug 2016 06:40:23 GMT" time="0.048" errors="0">
  <testcase name="factorialNegativeNumber" time="0.003" classname="bioinformatics_practice.rosalind.WrightFisherModelTest"/>
  <testcase name="factorialTest" time="0.002" classname="bioinformatics_practice.rosalind.WrightFisherModelTest"/>
  <testcase name="WFMoverGTest" time="0.033" classname="bioinformatics_practice.rosalind.WrightFisherModelTest">
    <failure type="junit.framework.ComparisonFailure" message="expected:&lt;...7...&gt; but was:&lt;...3...&gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<...7...> but was:<...3...>
	at junit.framework.Assert.assertEquals(Assert.java:81)
	at junit.framework.Assert.assertEquals(Assert.java:87)
	at bioinformatics_practice.rosalind.WrightFisherModelTest.WFMoverGTest(WrightFisherModelTest.java:17)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- WFMoverGTest -->
  <testcase name="calculateProbabilityTest" time="0.004" classname="bioinformatics_practice.rosalind.WrightFisherModelTest"/>
  <testcase name="factorialOne" time="0.002" classname="bioinformatics_practice.rosalind.WrightFisherModelTest"/>
  <testcase name="factorialZero" time="0.002" classname="bioinformatics_practice.rosalind.WrightFisherModelTest"/>
  <testcase name="wfmPositiveTest" time="0.002" classname="bioinformatics_practice.rosalind.WrightFisherModelTest"/>
</testsuite> <!-- bioinformatics_practice.rosalind.WrightFisherModelTest -->
